### DISCUSSION TITLE
https://github.com/orgs/community/discussions/122213#discussioncomment-9310156Account

### DISCUSSION THREAD

**Discussion by zuriel5zurielg20** [community/community#122213] [Code Security]
### Select Topic Area

Show & Tell

### Body

[com.android.chrome](url)

**Comment by zuriel5zurielg20** [comment]
> ### Seleccionar área temática
> Mostrar y contar
> 
> ### Cuerpo
> [com.android.cromo](url)

🤖

**Comment by ebndev** [comment]
Hi @zuriel5zurielg20,

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions as there is no context for the post. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

Please feel encouraged to post again when you have a question or a related topic you want to talk about.

Thank you!
